### DISCUSSION TITLE
Input() components which allow 'numeric' strings and update a ref-to-int or ref-to-float

### DISCUSSION THREAD

**Discussion by njk42** [ArthurSonzogni/FTXUI#464] [Ideas]
An Input() component which can be qualified (perhaps using a new InputOptions member) to only allow numeric inputs (i.e. filter its characters to only integers and the minus sign and decimal point) and then automatically update a ref-to-int or ref-to-float would be a valuable addition to this library.

**Comment by ArthurSonzogni** [comment]
That would be nice. However, this is a quite difficult problem. It depends on the user's locale. For instance, in french, we are using `13,3` for `13.3`
